meson.build

```diff
@@ -51,7 +53,7 @@ uenv_dep = [sqlite3_dep]
 
 uenv = executable('uenv',
         sources: uenv_src,
-        dependencies: [uenv_dep, lib_dep, fmt_dep, cli11_dep],
+        dependencies: [uenv_dep, lib_dep, fmt_dep, spdlog_dep, cli11_dep],
```

Comment:
This isn't needed, because `spdlog_dep` is transitive from `lib_dep`

Reply:
It's also a direct dependency of the cli tool, just like `fmt_dep` so should be declared explicitly as a dependency, no?

Reply:
Good point.
They should both be direct dependencies of the cli